client/app/states/requests/requests/details/details.html

```diff
@@ -24,6 +24,18 @@
                 <h2>{{ ::vm.request.description }}</h2>
                 <h4>{{ ::vm.request.options.long_description || vm.request.description }}</h4>
               </div>
+
+              <div ng-show="vm.pendingApproval()">
```

Comment:
`ng-if` preferred to `ng-show`  

reason, show essentially just plays with `display` if removes element from dom <3